src/components/AllArticles.jsx

```diff
@@ -0,0 +1,53 @@
+import Dropdown from "react-bootstrap/Dropdown";
+import { ArticleCard } from "./ArticleCard.jsx";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export const AllArticles = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setisLoading] = useState(true);
+
+  useEffect(() => {
+    axios
+      .get("https://news-api-40x5.onrender.com/api/articles")
+      .then(({ data: { articles } }) => {
+        setArticles(articles);
+        setisLoading(false);
+      });
+  }, []);
+
+  return (
+    <div className={"all-articles"}>
+      <div className={"all-articles-top"}>
+        <h2>All articles </h2>
+        <Dropdown>
+          <Dropdown.Toggle variant="danger" id="dropdown-basic">
+            Sort by
+          </Dropdown.Toggle>
+          <Dropdown.Menu>
+            <Dropdown.Item href="">Topic</Dropdown.Item>
+            <Dropdown.Item href="">Votes</Dropdown.Item>
+            <Dropdown.Item href="">Date</Dropdown.Item>
+          </Dropdown.Menu>
+        </Dropdown>
```

Comment:
Try to only add code required by the ticket. I know you have probably planned to have this but to mimic the real world of picking up tickets it is good practise tod only do what is asked. 👍